docs/source/examples/auto-failover.rst

```diff
@@ -3,29 +3,28 @@
 Auto-provisioning GPUs
 ==========================
 
+SkyPilot comes with an auto-failover provisioner, which
+**automatically retries provisioning** a cluster in different regions (or
+clouds) if the requested resources cannot be provisioned.
+Such failures can happen for a variety of reasons:
+
+- Out of capacity (in a region or a cloud)
```

Comment:
What's capacity? Consider:
`Out of capacity (all resources in a region or cloud are busy)`

Reply:
I considered this, but cannot find a better way to phrase it. ("all resources are busy" seems a bit inaccurate.)

Also considered
- Out of capacity (a region/cloud running out of the requested resources)
- Out of quotas (your account reaching the allowed limits of some resources in a region/cloud)

which seems way too long (and "running out" doesn't seem totally accurate). Let's defer this and assume knowledge with capacity/quotas for now.